tests/test.py

```diff
@@ -106,10 +106,7 @@ def test_mailto_bpdevs(page_url: tuple[Page, str]) -> None:
 
 @pytest.mark.parametrize(
     "url",
-    (
-        "/",
-        "/blog",
-    ),
```

Comment:
Removed this from our test case as it was causing the test to fail. The homepage no longer includes posts/blogs according to the latest changes and updates.